STEP/CodeGeneration/CodeGenerationVisitor.cs

```diff
@@ -93,10 +94,17 @@ private void EnterScope()
 
     private void CopyArrayHelper(ArrDclNode n) {
         // Copies array from RHS into LHS
-        var id = n.Right as IdNode;
         EmitIndentation();
+        var id = n.Right as IdNode;
         EmitLine($"memcpy({n.Left.Name}, {id.Name}, sizeof({id.Name}[0])*{Math.Min(n.Left.Type.ArrSize, id.Type.ArrSize)});");
     }
+    
+    private void CopyArrayHelper(ArrDclNode n, ArrLiteralNode lit) {
+        // Copies array from arr literal into LHS
+        EmitIndentation();
+        string tempName = $"temp{_tempCount - 1}";
```

Comment:
Det er umiddelbart lidt farligt at kalde den tempX, da det nok er brugt ret ofte af programmøren. Man kunne i hvert fald gøre det lidt bedre ved f.eks. at kalde den `__tempX__` eller sådan noget for at mindske chancen. Det gør de ofte i Python og C (behind the scenes)

Reply:
havde overvejet det, men nu hvor du selv foreslår det så gør jeg det!